templates/csharp/custom-copilot-weather-agent/appPackage/manifest.json.tpl

```diff
@@ -22,10 +22,19 @@
         "full": "Full description of {{appName}}"
     },
     "accentColor": "#FFFFFF",
+    "copilotAgents": {
+        "customEngineAgents": [
+            {
+                "type": "bot",
+                "id": "${{BOT_ID}}"
+            }
+        ]
+    },
     "bots": [
         {
             "botId": "${{BOT_ID}}",
             "scopes": [
+                "copilot",
                 "personal",
```

Comment:
I remember we get warning with 4 scopes here. Should we remove one scope?

Reply:
although there is a warning, but it can work actually
if we need to remove one. which one do you recommend

Reply:
Let's confirm with PM. I don't have special preference.